embedded/drivers/include/can.h

```diff
@@ -0,0 +1,56 @@
+#ifndef _CAN_H__
+#define _CAN_H__
+
+#include <linux/can.h>
+#include <net/if.h>
+#include <semaphore.h>
+#include <sys/ioctl.h>
+#include <sys/socket.h>
+#include <sys/time.h>
+#include <unistd.h>
+
+#include <cstring>
+#include <thread>
+#include <vector>
+
+#define NO_FILTER 0
+
+#ifdef USE_VCAN
+#define CAN_INTF "vcan0"
+#else
+#define CAN_INTF "can0"
+#endif
+
+static const struct itimerval new_val = {{0, 10000}, {0, 10000}};
```

Comment:
Let's drop this, since it isn't even used in pod-embedded. We can add it back if we ever feel like we need it

Reply:
by "this" I mean the `itimerval`

Reply:
no, we actually do use it on line 23 of can.cpp
 ```C
strcpy(this->ifr.ifr_name, CAN_INTF);
```
if opening a virtual can or not.

I suggest `grep -R CAN_INTF .` or my [fav tool](https://github.com/ggreer/the_silver_searcher)